process_mon.sh

```diff
@@ -64,4 +68,12 @@ if [ -z "$1" ]; then
 fi
 
 process_check=`ps ax | grep -v grep | grep -v process_mon | grep -c "$1"`
+if (( $process_check > 0 )); then
+  process_age=`ps -o etimes= -p "$process_check"`
```

Comment:
Since `$process_check` will contain either 0 or 1 from the `grep -c` this doesn't seem to grab the `etimes` from the intended pid.